### ISSUE TITLE
Differentiate path-not-found vs undetectable path type

### ISSUE THREAD

**Issue by alikhajeh1** [infracost/infracost#535]
The following two runs produce the same error, which confuses CI/CD users as they think the `path` param file/dir is present and something is wrong with the CLI. Usually in CI/CD, the issue is that the file cannot be found, because the path is wrong or, in the case of CircleCI, a previous task produced the plan.json file but this hasn't been attached to the infracost orb.

```
$ infracost breakdown --path nonexistentfile
Error: Could not detect path type for nonexistentfile
...

$ echo abc > plan.json
$ infracost breakdown --path plan.json
Error: Could not detect path type for plan.json
```

It would be clearer if the first error was "Error: No such file or directory plan.json" (similar to what `ls nonexistentdir` outputs).

**Comment by Pauluson** [comment]
i want to do it

**Comment by alikhajeh1** [comment]
Released in v0.8.4
